api/translate_names/translate_names.py

```diff
@@ -0,0 +1,36 @@
+from api import util
+
+
+def translate_names_main(name_type, name, name_type2, major=None, minor=None):
```

Comment:
this all looks decidedly odd to me... 
```suggestion
def translate_names_main(input_type, major, minor):
```

my standard name translation algorithm would be something like:
1. process `input_type` to lowercase
2. use regex to remove unexpected nonsense like non-word characters
3. translate via dict e.g. 
```
translate = {
  "distgit": "distgit",
  "dist_git": "distgit",
  ...
}
(and complain if it's not there of course)

Reply:
I copied it exactly from art-bot, but I actually don't know what it does... maybe I should not include that in this PR, since its not exactly needed in this front end. I'll add it here once art-dash starts to need it.